MaterialLib/Fluid/Viscosity/WaterViscosityIAPWS.cpp

```diff
@@ -0,0 +1,172 @@
+/**
+ *  \copyright
+ *   Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *              Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   WaterViscosityIAPWS.cpp
+ *
+ * Created on December 1, 2016, 1:41 PM
+ */
+
+#include "WaterViscosityIAPWS.h"
+
+#include <cmath>
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+const double WaterViscosityIAPWS::_hi[4] = {1.67752, 2.20462, 0.6366564,
+                                            -0.241605};
+const double WaterViscosityIAPWS::_hij[6][7] = {
```

Comment:
According the concept of OOP, the constants should be the members of the class because they are the data of the this model. However, since this class is not inheritable, moving the constants to the cpp does not break the data encapsulation and it makes the head file concise as well. Therefore I changed the implematation according to your suggestion. Thanks.